src/test-utils/smoke-test-utils.ts

```diff
@@ -56,7 +104,8 @@ export const verifyIngestionWithRetry = async (
                 rumClient,
                 eventIds,
                 timestamp,
-                monitorName
+                monitorName,
+                metadataAttributes
             );
             return true;
```

Comment:
suggestion: Refactor this method so that each step (getting data and verifying data) is implemented as a separate function. We could also further decompose the verification step.
```javascript
const actual: Map<string, string[]> = await getIngestedEvents(
    rumClient,
    eventIds,
    timestamp,
    monitorName,
    metadataAttributes
);
return expectEvents(actual)
    && expectValidAttributes(actual)
    && expectInvalidAttributes(actual);
```